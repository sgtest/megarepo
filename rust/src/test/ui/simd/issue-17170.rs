#![feature(repr_simd)]

#[repr(simd)]
struct T(f64, f64, f64);
//~^ ERROR SIMD vector length must be a power of two

static X: T = T(0.0, 0.0, 0.0);

fn main() {
    let _ = X;
}
