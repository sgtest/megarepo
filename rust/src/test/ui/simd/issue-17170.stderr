error[E0075]: SIMD vector length must be a power of two
  --> $DIR/issue-17170.rs:4:1
   |
LL | struct T(f64, f64, f64);
   | ^^^^^^^^^^^^^^^^^^^^^^^^

error: monomorphising SIMD type `T` of non-power-of-two length

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0075`.
